Makefile

```diff
@@ -1,3 +1,11 @@
+VERSION := $(shell git describe --tags)
```

Comment:
Should we perhaps run `git fetch --tags` before, in case tags are not up-to-date in developer's machine?